### ISSUE TITLE
How to force display:none with !important on Week and Year ?

### ISSUE THREAD

**Issue by aalfiann** [elreco/vue-tailwind-datepicker#155]
**Describe the bug**
This vue-tailwind-datepicker is works well but I got some problem on Week and Year display.  
This is might be a bug only for me because of my css template. Because my app is already big, I can't just modify the css.

it doesnt hide the Week and Year. so I got to use style="display:none !important" on element with class "px-0.5 sm:px-2" which is contains "flex flex-wrap"

Here is the screenshot
![](https://i.imgur.com/bcVeWsn.png)

**Comment by artytechy** [comment]
The same problem is happening to me, but I do not have any custom CSS templates. I attempted to remove DaisyUI, but the problem persisted. Have you resolved the issue?

**Comment by aalfiann** [comment]
> The same problem is happening to me, but I do not have any custom CSS templates. I attempted to remove DaisyUI, but the problem persisted. Have you resolved the issue?

Yes I did, there is workaround using javascript to handle this, but the performance is bad. Better to solve this in Vue way.

**Comment by artytechy** [comment]
My issue was resolved when I attempted to delete or set to `false` `important: true` in the tailwind.config.js file.

**Comment by aalfiann** [comment]
Yeah, I did it in the first time, but it gonna broke some of element in my website. since this is a big website, I can't do that.
